### PR TITLE
Updated text

### PR DESCRIPTION
**Pull request by ellenbechtel**
Before making a pull request
----------------------------
First . . .
- [x] Clean the code the way Vue likes it - run 'npm run lint --fix'        
- [ ] Make sure all tests run

Then check for accessibly compliance
- [ ] Run WAVE plugin 508 compliance tool

Then run Browserstack; check that application works on . . .
- [ ] Chrome
- [ ] Safari
- [ ] Edge
- [x] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

Finally . . .
- [ ] Update the changelog appropriately

Updated all text
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: src/views/3WaterSupply/WaterSupply.vue [hunk 1]
```diff
@@ -63,9 +62,8 @@
...
+     <p>This rapidly moving water can pick up and <a href="https://iopscience.iop.org/article/10.1088/1748-9326/10/8/084007">carry the wildfire ash and sediment into streams.</a> Heavy rains can even lead to destructive <a href="https://landslides.usgs.gov/hazards/postfire_debrisflow/">debris flows.</a> Formerly clean, clear streams become clogged with ash, sediment, and burned materials. In forests that have been impacted by other disturbances, such as air pollution or mining, nutrient and <a href="https://www.sciencedirect.com/science/article/pii/S0048969720341577?via%3Dihub">metal concentrations</a> can also increase as the overland flow mobilizes more material. These watershed changes pose challenges water managers, who then must use additional treatments and incur extra costs before this water can be used for municipal water supply.</p>
```
Comment (hcorson-dosch-usgs) [inline src/views/3WaterSupply/WaterSupply.vue]: One small typo here:
>These watershed changes pose challenges **for** water managers, who then must use additional treatments and incur extra costs before this water can be used for municipal water supply

File: src/views/5NGWOS/NGWOS.vue [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        <p>The USGS <a href="to https://www.usgs.gov/mission-areas/water-resources/science/usgs-next-generation-water-observing-system-ngwos?qt-science_center_objects=0#qt-science_center_objects">Next Generation Water Observing System</a> (NGWOS) is generating integrated data on streamflow, groundwater, evapotranspiration, snowpack, soil moisture, water quality, and water use. </p>
+        <p>One focus of NGWOS will be the <a href="https://www.usgs.gov/mission-areas/water-resources/science/next-generation-water-observing-system-upper-colorado-river?qt-science_center_objects=0#qt-science_center_objects">Upper Colorado River Basin,</a> an area that provides much of the flow in the Colorado River. The Colorado River  supplies water for more than 40 million people and nearly 5.5 million acres of farmland across the western United States and Mexico. Yet the Upper Colorado River Basin is vulnerable to drought, wildfire, changing snowpack, and many other factors that affect water supply. The USGS WMA is working to improve measurement and modeling of wildfire impacts on water availability in the region, which was affected by major wildfires in 2020. </p>
```
Comment (hcorson-dosch-usgs) [inline src/views/5NGWOS/NGWOS.vue]: Tiny typo - extra space between "River" and "supplies" in second sentence

File: src/views/Resources/Resources.vue [hunk 3]
```diff
@@ -4,7 +4,8 @@
     class="text-content"
   >
     <h2><span class="lowlight">Further Research</span></h2>
-    <ul>
+    <p>For more information about USGS work on wildfires and water supplies, visit the <a href="https://www.usgs.gov/special-topic/fire">Wildland Fire Science</a>, the <a href="https://ca.water.usgs.gov/wildfires/wildfires-water-quality.html">California Water Science Center,</a> or any of the USGS peer-reviewed research on wildfires, some of which is listed below.</p>
```
Comment (hcorson-dosch-usgs) [inline src/views/Resources/Resources.vue]: Small thing here, reads
>For more information about USGS work on wildfires and water supplies, visit the Wildland Fire Science, the California Water Science Center, or any of the USGS peer-reviewed research on wildfires, some of which is listed below.

That first 'the' before 'Wildland' feels weird. Maybe say 'consult Wildland Fire Science' or 'consult the Wildland Fire Science page'?

File: src/views/3WaterSupply/WaterSupply.vue (hunk 1 above)
Comment (ellenbechtel) [reply src/views/3WaterSupply/WaterSupply.vue]: good catch!

File: src/views/0VizHeader/Header.vue [hunk 4]
```diff
@@ -4,19 +4,52 @@
...
-        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 303.96 154.98">
-          <g  transform="translate(120 80)" id="burn_2020" >
-              <rect id="box-2020" width="150" height="60" style="fill: rgb(245,169,60)"/>
-              <text id="text-2020" transform="translate(7.55 19.61)" style="font-size: 1.2em; font-weight: 200" >Over 8 million acres <tspan x="0" y="15">have burned in 2020, <tspan x="0" y="30">the most on record</tspan></tspan></text>
-              <g id="arrow-2020" transform="translate(155 35)">
-                 <path d="M23.54,37.23A60.65,60.65,0,0,0,3.09.45C1.37-1-1.15,1.47.58,3A57.73,57.73,0,0,1,20.13,38.17a1.79,1.79,0,0,0,2.18,1.24,1.81,1.81,0,0,0,1.23-2.18Z" style="fill: rgb(245,169,60)"/>
-                <path d="M25.51,37.91,26.86,21.7a2,2,0,0,0-4,0L21.47,37.91a2,2,0,0,0,4,0Z" style="fill: rgb(245,169,60)"/>
-                <path d="M21.66,35.31c-4.93-.6-9.89-1-14.82-1.63a2,2,0,0,0-2,2,2.07,2.07,0,0,0,2,2c4.93.6,9.89,1,14.82,1.63a2,2,0,0,0,2-2,2.07,2.07,0,0,0-2-2Z" style="fill: rgb(245,169,60)"/>
-                </g>
+        <svg
```
Comment (cnell-usgs) [inline src/views/0VizHeader/Header.vue]: It's saying there is a conflict in Header.vue but if this is the only change, it looks fine to me

**Comment by cnell-usgs** [comment]
I think the merge conflicts might be resolved by pulling down the most recent version of the master branch and pushing that to this PR

**Comment by ellenbechtel** [comment]
@collnell you were right, just pulling down the upstream changes fixed it!  I made a few header sizing tweaks while i was in there to make sure the annotation was visible.

**Pull request merged**